src/Actuators/WebChat/WebChatListElement.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace actsmart\actsmart\Actuators\WebChat;
+
+class WebChatListElement
+{
+    private $title = null;
+
+    private $subTitle = null;
+
+    private $image = null;
+
+    private $buttonText = null;
+
+    private $buttonCallback = null;
+
+    private $buttonUrl = null;
+
+    private $buttonLinkNewTab = true;
+
+    /**
+     * @param $title
+     * @param $subTitle
+     * @param $image
+     * @param $buttonText
+     * @param $buttonCallback
+     * @param $buttonUrl
+     * @param $buttonLinkNewTab
+     */
+    public function __construct($title, $subTitle, $image, $buttonText, $buttonCallback, $buttonUrl, $buttonLinkNewTab = true)
```

Comment:
Can we take the buttonText, buttonCallback ,buttonUrl and buttonLinkNewTab out of the constructor and add 2 new methods:

addCallbackButton($buttonText, $buttonCallback)

addLinkButton($buttonText, $buttonUrl, $buttonLinkNewTab = true)

to make it cleaner